entrypoint.sh

```diff
@@ -4,12 +4,21 @@ image=$1
 report=$2
 patched_tag=$3
 timeout=$4
+output_file=$5
 
 # parse image into image name
 image_no_tag=$(echo "$image" | cut -d':' -f1)
 
+# check if output_file has been set
+if [ -z "$output_file" ]
+then
+    output=""
+else
+    output="--format=openvex --output $output_file"
+fi
+
 # run copa to patch image
-if copa patch -i "$image" -r ./data/"$report" -t "$patched_tag" --addr tcp://127.0.0.1:8888 --timeout $timeout;
```

Comment:
can we add tests for this please?

https://github.com/project-copacetic/copa-action/blob/main/.github/workflows/build.yaml#L53-L55
https://github.com/project-copacetic/copa-action/blob/main/test/test.bats

Reply:
Test added